src/OpenSheetMusicDisplay/Cursor.ts

```diff
@@ -167,4 +169,21 @@ export class Cursor {
   public get Hidden(): boolean {
     return this.hidden;
   }
+
+  public get AllVoicesUnderCursor(): VoiceEntry[] {
+    return this.VoicesUnderCursor();
+  }
+
+  public VoicesUnderCursor(instrument?: Instrument): VoiceEntry[] {
+    return this.iterator.CurrentVisibleVoiceEntries(instrument);
+  }
+
+  public NotesUnderCursor(instrument?: Instrument): Note[] {
+    const voiceEntries: VoiceEntry[]  = this.VoicesUnderCursor(instrument);
+    const notes: Note[] = [];
+    voiceEntries.forEach(voiceEntry => {
+      notes.push.apply(notes, voiceEntry.Notes);
+    });
+    return notes;
+  }
```

Comment:
this method is a bit more complex, so while not strictly necessary, if you use it often, it makes sense to put it into a method.
On the other hand, OSMD itself doesn't need these utility methods, so one could ask why we need it in OSMD and people couldn't just implement this as a static utility function themselves.
But we've seen multiple requests for more Cursor utility functions, so why not.

Reply:
You're right. I didn't realize everything necessary is now available via public getters. One could just write something like:

```typescript
function getVoicesUnderCursor (instrument?: Instrument): VoiceEntry[] {
    osmd.cursor.Iterator.CurrentVisibleVoiceEntries(instrument)
}

function getNotesUnderCursor (instrument?: Instrument) {
    const voiceEntries: VoiceEntry[]  = this.VoicesUnderCursor(instrument);
    const notes: Note[] = [];
    voiceEntries.forEach(voiceEntry => {
      notes.push.apply(notes, voiceEntry.Notes);
    });
    return notes;
}
```

However, the above definitions would fail because `Instrument`, `VoiceEntry` and `Note` are not exported AFAIK. To define these methods, we need access to the `Instrument` type, but I'd also export `Note` and `VoiceEntry`, because without it we'd have to solely rely on TypeScript's type inference when calling these methods. For example, we'd be able to write:

```typescript
const notes = getNotesUnderCursor()
```

but not:

```typescript
const notes: Note[] = getNotesUnderCursor()
```

Reply:
actually, these are exported now.